libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -382,6 +382,12 @@ void AP_MotorsHeli_RSC::update_rotor_runup(float dt)
     if (_runup_complete && (get_rotor_speed() <= get_critical_speed())) {
         _runup_complete = false;
     }
+    // if rotor ramp and runup are both at zero, then spooldown has been completed
+    if (get_rotor_speed() <= 0.0f) {
```

Comment:
you could of course shorten this to a single line but it's a style thing..

Reply:
Done.  Thanks!